ipmininet/cli.py

```diff
@@ -81,6 +81,18 @@ def do_ping6pair(self, _line):
         """Ping (IPv6-only) between first two hosts, useful for testing."""
         self.mn.ping6Pair()
 
+    def do_link(self, line):
+        """ down/up the link between 2 specified routers, can specify multiple
+            multiple link 
```

Comment:
Could you describe the format of line here ?

Reply:
Yes i didn't do it because it's not something which is done in the other things